ext/random/engine_pcgoneseq128xslrr64.c

```diff
@@ -153,7 +153,7 @@ PHP_METHOD(Random_Engine_PcgOneseq128XslRr64, __construct)
 	ZEND_PARSE_PARAMETERS_END();
 
 	if (seed_is_null) {
-		if (php_random_bytes_throw(&state->state, sizeof(php_random_uint128_t)) == FAILURE) {
+		if (php_random_bytes_throw(&state->state, sizeof(state->state)) == FAILURE) {
```

Comment:
I'm assuming this is determined at compile time because the compiler knows the struct field type.

Reply:
`sizeof` is _always_ compile time.

Reply:
That is *technically* incorrect `sizeof` of a VLA is a runtime construct. :)

Reply:
TIL, but VLA are terrible anyway 😄